docs/getting_started.md

```diff
@@ -157,6 +169,45 @@ quay:
   access_token: 'YOUR_TOKEN'
 ```
 
+## Deploying Clusters and Running Tests
+
+Once you have set up the configuration files you are ready to start
+executing the code included in the ocs-ci. First you must prepare
+your environment for running the scripts:
+
+* Run: `python setup.py develop`
```

Comment:
We are mentioning the way of using virtual env - this can maybe confuse the peoples here.  Once you setup venv you can then just activate again and you will have everything in your path, why we need to mention here this approach?

Reply:
Just creating a virtualenv doesn't create a runnable 'run-ci' command. Unless I missed some sort of magic when I was trying to get the command running the first time.

Reply:
You need to activate your virtual by source ./path_to_venv/bin/activate  ;). it's described int he readme ;) 

It does the magic as it add the ./venv/bin directory to your path where the executable file is.

```
pbalogh@MacBook-Pro ocs-ci (lib-bucket-provisioner-wa-with-kcs-wa) $ run-ci
-bash: run-ci: command not found
pbalogh@MacBook-Pro ocs-ci (lib-bucket-provisioner-wa-with-kcs-wa) $ source .venv/bin/activate
(.venv) pbalogh@MacBook-Pro ocs-ci (lib-bucket-provisioner-wa-with-kcs-wa) $ echo $PATH
/Users/pbalogh/Code/OCS/ocs-ci/.venv/bin:...omitted...
(.venv) pbalogh@MacBook-Pro ocs-ci (lib-bucket-provisioner-wa-with-kcs-wa) $ ls .venv/bin/
__pycache__/           f2py3*                 jsonschema*            pyrsa-decrypt*         rst2latex.py*
activate               f2py3.7*               markdown_py*           pyrsa-encrypt*         rst2man.py*
activate.csh           futurize*              netaddr*               pyrsa-keygen*          rst2odt.py*
activate.fish          get_objgraph*          nova*                  pyrsa-priv2pub*        rst2odt_prepstyles.py*
aws-cleanup*           google-oauthlib-tool*  openshift-ansible-gen* pyrsa-sign*            rst2pseudoxml.py*
bugzilla*              hcltool*               pasteurize*            pyrsa-verify*          rst2s5.py*
chardetect*            ipdb3*                 pbr*                   pytest*                rst2xetex.py*
ci-cleanup*            ipmitool.py*           pip*                   python@                rst2xml.py*
ci-pause*              iptest*                pip3*                  python3@               rstpep2html.py*
cinder*                iptest3*               pip3.7*                report-version*        run-ci*
easy_install*          ipython*               py.test*               rst2html.py*           undill*
easy_install-3.7*      ipython3*              pygmentize*            rst2html4.py*          wsdump.py*
f2py*                  jp.py*                 pyjwt*                 rst2html5.py*
(.venv) pbalogh@MacBook-Pro ocs-ci (lib-bucket-provisioner-wa-with-kcs-wa) $ which run-ci
/Users/pbalogh/Code/OCS/ocs-ci/.venv/bin/run-ci
```

Reply:
I think we're talking past each other. I did miss a bit of magic. But the magic is done by the pip command not by the virtualenv itself. Manually running 'setup.py develop' it not needed because the "-e" flag inside the requirements.txt does it automatically. It strikes me as a little odd that the project thus becomes it's own dependency but that can be an item for another day. I can drop this extra step (for now).